classes/utils/Utilities.class.php

```diff
@@ -385,7 +385,7 @@ public static function sanitizeInput($input) {
             
             // Render potential tags useless by putting a space immediatelly after < which does not already have one
             $input = html_entity_decode($input, ENT_QUOTES, 'UTF-8');
-            $input = preg_replace('`<([^\s])`', '< $1', $input);
+            $input = preg_replace('`<([^\S])`', '< $1', $input);
```

Comment:
Are you sure this is correct? I think changing `\s` to `\S` switches around the meaning, doesn't it? Maybe you wanted to change `[^\s]` to `\S`?

Reply:
indeed the change of \s to \S change is a mistake :)